server.js

```diff
@@ -0,0 +1,3 @@
+import init from './src/backend/index'
+const shouldStart = true
```

Comment:
This file should be `index.js` instead. Also there is no need for this variable as the value `true` can be passed directly into the `init` function call.